24.x-rebase: Vertcoin Core Branding

cruelnovo opened the pull request:
This PR includes all of the branding updates for Vertcoin Core.  The man pages have been removed from branding - those will be created by the release manager during release build process.

cruelnovo committed:
vtc branding: .github folder

cruelnovo committed:
vtc branding: share folder

cruelnovo committed:
vtc branding: util folder

cruelnovo committed:
vtc branding: wallet folder

vertiond commented on configure.ac:
```diff
@@ -2,12 +2,12 @@ AC_PREREQ([2.69])
-define(_COPYRIGHT_YEAR, 2022)
+define(_CLIENT_VERSION_RC, 1)
+define(_CLIENT_VERSION_IS_RELEASE, false)
+define(_COPYRIGHT_YEAR, 2023)
```
Leave these for the release manager

cruelnovo replied:
I backed out the changes to the release version and copyright year

vertiond commented:
Unique things like our copyright string could be separated out from https://github.com/vertcoin-project/vertcoin-core/pull/252/commits/ff09de5fe5f973617c642af244b7950c2a000a2e 
ie: https://github.com/vertcoin-project/vertcoin-core/commit/5647a53c13d9fb05fb2314ab80bfcd891f36f0d4

vertiond commented on src/qt/splashscreen.cpp:
```diff
@@ -43,7 +43,8 @@ SplashScreen::SplashScreen(const NetworkStyle* networkStyle)
     QString versionText     = QString("Version %1").arg(QString::fromStdString(FormatFullVersion()));
-    QString copyrightText   = QString::fromUtf8(CopyrightHolders(strprintf("\xc2\xA9 %u-%u ", 2009, COPYRIGHT_YEAR)).c_str());
+    QString copyrightText   = QString::fromUtf8(CopyrightHolders(strprintf("\xc2\xA9 %u-%u ", 2014, COPYRIGHT_YEAR),
+                                                                 strprintf("\xc2\xA9 %u-%u ", 2009, COPYRIGHT_YEAR)).c_str());
```
Copyright string being updated in different commits. This unique item should be updated in one, clear commit.

vertiond commented on doc/man/Makefile.am:
```diff
@@ -1,27 +1,27 @@
 if ENABLE_WALLET
 if BUILD_BITCOIN_WALLET
-  dist_man1_MANS+=vertcoin-wallet.1
+  dist_man1_MANS+=bitcoin-wallet.1
```
What is this commit?

cruelnovo replied:
I corrected the commit by removing the manpage changes from 'vtc branding'.  Building the manpages needs to be a part of the release process.

vertiond commented on contrib/windeploy/win-codesign.cert:
```diff
@@ -1,112 +1,131 @@
+FHxOU09fliveaTiE4fJvShu6yf56N9Tf5VZZDTf6yspR1j5hNsg18V9zvzslWixQ
+3+TO1McMFOJw4PxH2HOupRw86dqWaOS0TvU=
+-----END CERTIFICATE-----
+
```
This cert is actually invalid. Let's leave this alone while we don't have our own cert.

cruelnovo replied:
Changes to 'win-codesign.cert' have been backed out of this commit.

cruelnovo committed:
vtc branding: README, make, and build

Add Vertcoin icons for Vertcoin Branding

cruelnovo committed:
vtc branding: RPC

vertiond commented on src/Makefile.am:
```diff
@@ -123,6 +123,7 @@ BITCOIN_CORE_H = \
   banman.h \
   base58.h \
   bech32.h \
+  bignum.h \
```
nit: this should only need to be [here](https://github.com/vertcoin-project/vertcoin-core/pull/248/files#diff-4cb884d03ebb901069e4ee5de5d02538c40dd9b39919c615d8eaa9d364bbbd77R126)

cruelnovo replied:
Removed the addition of bignum.h in this 'branding' commit.

vertiond commented on src/clientversion.cpp:
```diff
@@ -77,23 +77,19 @@ std::string FormatSubVersion(const std::string& name, int nClientVersion, const
 }
 
-std::string CopyrightHolders(const std::string& strPrefix)
+std::string CopyrightHolders(const std::string& strPrefix1, const std::string& strPrefix2)
```
I think it would still make sense for our copyright string changes to be in a separate commit

cruelnovo replied:
I agree.  I've created another commit as part of this PR that has all copyright changes for Vertcoin

cruelnovo committed:
vtc branding: QT

cruelnovo committed:
vtc branding: src folder

cruelnovo committed:
vtc branding: contrib folder

Should include changes for branding; excludes changes to be made by maintainer when producing a release (build.sh, etc)

cruelnovo committed:
vtc branding: add copyright string for Vertcoin developers

vertiond commented on doc/README.md:
```diff
@@ -63,14 +72,14 @@ The Bitcoin repo's [root README](/README.md) contains relevant information on th
 
 ### Miscellaneous
 - [Assets Attribution](assets-attribution.md)
+- [Assumeutxo design](assumeutxo.md)
```
This doesn't exist

vertiond replied:
it's trivial but a sign to be careful about cherry-picking

cruelnovo replied:
Great catch! This is fixed - the Assumeutxo design now points to the correct file (design/assumeutxo.md)

vertiond commented on doc/README.md:
```diff
@@ -63,14 +72,14 @@ The Bitcoin repo's [root README](/README.md) contains relevant information on th
 - [BIPS](bips.md)
 - [Dnsseed Policy](dnsseed-policy.md)
 - [Benchmarking](benchmarking.md)
-- [Internal Design Docs](design/)
```
This exists

cruelnovo replied:
Added this documentation reference back in to the README.md file.

cruelnovo committed:
vtc branding: doc folder
